definitions/grib2/marsLevtypeConcept.def

```diff
@@ -13,13 +13,15 @@
 'pl'   = {typeOfFirstFixedSurface=100; typeOfSecondFixedSurface=255;}
 'pl'   = {typeOfFirstFixedSurface=100; typeOfSecondFixedSurface=100;}
 'sfc'  = {typeOfFirstFixedSurface=101; typeOfSecondFixedSurface=255;}
-'sfc'  = {typeOfFirstFixedSurface=102; typeOfSecondFixedSurface=255;}
+# heightAboveSea
+'ml'   = {typeOfFirstFixedSurface=102; typeOfSecondFixedSurface=255;}
 'sfc'  = {typeOfFirstFixedSurface=103; typeOfSecondFixedSurface=255;}
```

Comment:
I would introduce antother key value 'zl' for this level type. I am planning to add also a height above ground level type 'hl', with rules for special meteorologocal parameters as 2m-temperature, 10m wind components, as ECMWF introduced for Destination Earth in another PR.